.github/workflows/ci.yml

```diff
@@ -37,6 +37,13 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install protobuf==3.7.1
+    
+    # Chrome Stable is already installed by default.
+    - name: Install Chrome ToT
+      run: bash lighthouse-core/scripts/download-chrome.sh
```

Comment:
AFAIK there's no caching w/ this approach. I don't think we want to cache ToT installation, right?

Reply:
I could see us caching within the same day or something, but I agree we don't want to cache for very long. either way followup seems fine.